chihacknight/decarbonize-my-state
# state detail placeholder images, links & polish

## Overview

This PR takes a pass on the state detail page by adding images from the wireframes and a few additional polishes

* adds placeholder images for gas appliances, electric appliances, electric cars and power plant transitions on the state detail page
* shows the raw number of mmtco2e to reduce each year for each state instead of a static 3.6%
* adds links to some key phrases like electric heat pumps and induction ranges
* adds county name under each power plant to help identify location (in lieu of a map)
* adds logic to hide text and image list for power plants when the `length` is `0`
* a few smaller text changes

Relates to #57 

### Demo

![Screen Shot 2022-05-04 at 9 39 09 PM](https://user-images.githubusercontent.com/919583/166856334-1a9a7026-c509-4a12-8fa9-35d95bfd7aab.png)

## Testing Instructions

* check several state detail pages and confirm the images show up where they should
* for states with no plants of a certain type (rhode_island has no coal plants, for example) confirm the text and list do not show
